ssl-impl/src/test/java/io/crate/protocols/postgres/SslReqHandlerTest.java

```diff
@@ -120,8 +114,10 @@ public void testClassLoadingFallback() {
         }
     }
 
-    @Test(expected = SslConfigurationException.class)
+    @Test
     public void testClassLoadingWithInvalidConfiguration() {
+        expectedException.expect(SslConfigurationException.class);
+        expectedException.expectMessage("Failed to build SSL configuration");
```

Comment:
thats exactly what we're up to: the test shall fail if we change the message as this is a change which must be announced. pls always use this instead of the annotation